server/api/schedule.ts

```diff
@@ -0,0 +1,200 @@
+import { GET_EVENT_BY_ID } from '~~/queries/get-event-by-id';
+import useSwapcardClient from '../useSwapcardClient';
+import { groupBy, uniqBy } from 'es-toolkit';
+
+interface Speaker {
+    id: string;
+    firstName: string;
+    lastName: string;
+    email: string;
+    photoUrl: string;
+    organization: string;
+    jobTitle: string;
+}
+
+interface Session {
+    id: string;
+    beginsAt: string;
+    endsAt: string;
+    categories: string[];
+    title: string;
+    place: string;
+    type: string;
+}
+
```

Comment:
Il manquerait une interface de plus ici afin que ça soit type safe avec Typescript:
```suggestion
interface GroupedSession {
    id: string;
    startAt: string;
    endsAt: string;
    categories: string[];
    title: string;
    place: string;
    type: string;
    speaker: Speaker;
}
```